packages/core/src/rest/ZosmfHeaders.ts

```diff
@@ -115,7 +122,7 @@ export class ZosmfHeaders {
      * @static
      * @memberof ZosmfHeaders
      */
-    public static readonly X_CSRF_ZOSMF_HEADER: object = { "X-CSRF-ZOSMF-HEADER": true }; // "the value does not matter"
+    public static readonly X_CSRF_ZOSMF_HEADER: object = { "X-CSRF-ZOSMF-HEADER": "true" }; // "the value does not matter"
```

Comment:
I assume this change was made to match the `IHeaderContent` type which defines all properties as strings 🙂 - should we change the `object` type to `IHeaderContent` to reflect this?